src/about/conduct.html

```diff
@@ -106,7 +106,32 @@ <h2>Local authorities</h2>
 {% markdown %}
 ## Procedures for Reporting Incidents
 
-If you believe someone has violated the Code of Conduct, we encourage you to report it. If you are unsure whether the incident is a violation, or whether the space where it happened is covered by the Code of Conduct, we encourage you to still report it. We are fine with receiving reports where we decide to take no action for the sake of creating a safer space.
+If you believe someone has violated the Code of Conduct, we encourage you to report it.
+
+## Examples of Inappropriate Behavior
+-   Harassment of any participants in any form
+-   Deliberate intimidation, stalking, or following
+-   Logging or taking screenshots of online activity for harassment purposes
+-   Publishing others’ private information, such as a physical or electronic address, without explicit permission
+-   Violent threats or language directed against another person
+-   Incitement of violence or harassment towards any individual, including encouraging a person to commit suicide or to engage in self-harm
+-   Creating additional online accounts in order to harass another person or circumvent a ban
+-   Sexual language and imagery in online communities or in any conference venue, including talks
+-   Insults, put downs, or jokes that are based upon stereotypes, that are exclusionary, or that hold others up for ridicule
+-   Excessive swearing
+-   Unwelcome sexual attention or advances
+-   Unwelcome physical contact, including simulated physical contact (eg, textual descriptions like “hug” or “backrub”) without consent or after a request to stop
+-   Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+-   Sustained disruption of online community discussions, in-person presentations, or other in-person events
+-   Continued one-on-one communication after requests to cease
+-   Other conduct that is inappropriate for a professional audience including people of many different backgrounds
+
+This list is not an exhaustive list of all inappropriate behavior that is unacceptable at DjangoCon US.
+
+If you are unsure whether the incident is a violation, or whether the space where it happened is covered by the Code of Conduct, we encourage you to still report it. 
+
+We are fine with receiving reports where we decide to take no action for the sake of creating a safer space.
```

Comment:
I'd prefer to see this above the list of examples so the person knows that before they try finding a relatable scenario.

Reply:
checking on that in a minute